package.json

```diff
@@ -1,26 +1,49 @@
 {
   "name": "saa_victoria_subtheme",
-  "version": "1.0.0",
+  "version": "8.1.6",
```

Comment:
This version should now match and stay in sync with the Drupal theme version and the tag/release version.